makeflow/test/TR_makeflow_json_local.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+. ../../dttools/test/test_runner_common.sh
+
+prepare()
+{
+	clean $@
+}
+
+run()
+{
+	# slurm shouldn't be set up here, so this should only run
+	# if local_job works
+	cd json && ../../src/makeflow -T slurm --json local.json && exit 0
```

Comment:
I like this as a means to ensure local is working. Would there be any point in having a noop batch system for testing things like this?